shared_model/interfaces/common_objects/account_asset.hpp

```diff
@@ -81,7 +81,8 @@ namespace shared_model {
         oldModel->account_id = accountId();
         oldModel->asset_id = assetId();
         using OldBalanceType = decltype(oldModel->balance);
-        /// Use shared_ptr and placement-new to copy new model field to oldModel's field and
+        /// Use shared_ptr and placement-new to copy new model field to
+        /// oldModel's field and
```

Comment:
Merge this comment with comment below, otherwise looks strange, and also:
> to **the field of** oldModel ~~'s field~~ and ~~to~~ **for** return**ing** raw pointer